services/web/server/src/simcore_service_webserver/utils.py

```diff
@@ -196,3 +196,8 @@ def get_tracemalloc_info(top=10) -> List[str]:
         )
 
     return top_trace
+
+
+def compose_error_msg(msg: str) -> str:
+    msg = msg.strip()
+    return f"{msg}. Please send this message to support@osparc.io [{now_str()}]"
```

Comment:
where will this pop up?

Reply:
in the browser since ``/study/`` returns text content. I am creating nice error site template (check [these](https://colorlib.com/wp/free-404-error-page-templates/) )